src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,9 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        User user;
+        UserService userService = new UserService();
+        user = userService.findByEmail(email);
```

Comment:
`userService` should be class-level `private final` field. no need to create it each time when we call this method
```suggestion
        User user = userService.findByEmail(email);
```

Reply:
.